.github/actions/spelling/allow.txt

```diff
@@ -147,6 +147,7 @@ dbhost
 dbpwd
 dbuser
 debian
+dependant
```

Comment:
I think it should be dependent not dependant.
Also, later it may likely be initializing not initialising: https://dictionary.cambridge.org/dictionary/english/initializing

Reply:
Aren't we using UK english? https://www.merriam-webster.com/grammar/spelling-variants-dependent-vs-dependant